camayoc/tests/qpc/api/v1/conftest.py

```diff
@@ -88,5 +88,13 @@ def run_scan(
 
 
 def scan_list():
-    """Generate list of scan dict objects found in config file."""
-    return settings.scans
+    """Generate list of netwok / VCenter / Satellite scan objects found in config file."""
+    scans = []
+    supported_source_types = ("network", "vcenter", "satellite")
+    source_names_types = {s.name: s.type for s in settings.sources}
+    for scan in settings.scans:
+        source_types = [source_names_types.get(source) for source in scan.sources]
+        if not all(source_type in supported_source_types for source_type in source_types):
+            continue
```

Comment:
if for some reason we get a combination like "ansible" + "vcenter", this would be skipped. Is that ok?
(Probably fine, right? I believe our tests here usually have scans with a single source)

Reply:
For now all our scans use sources of single type, so I think this is OK.

In general, I would like to have more useful fixtures. I'm think about things like "all scans", "all scans of type X", "all scans with sources of different type". The main problem is that I don't yet have good intuition how exactly to model that, so this is readable, maintainable and useful.